packaging/rpm/microshift.spec

```diff
@@ -445,6 +453,19 @@ install -p -m644 packaging/tuned/runtime-class/kustomization.yaml %{buildroot}/%
 install -p -m644 packaging/tuned/microshift-tuned.service %{buildroot}%{_unitdir}/microshift-tuned.service
 install -p -m755 packaging/tuned/microshift-tuned.py %{buildroot}%{_bindir}/microshift-tuned
 
+# gateway-api
+install -d -m755 %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-gateway-api
```

Comment:
Does it have to 000? Ideally each manifest would have different number

Reply:
I understand these prefixes as "dependency ordering". Since gateway api does not depend on anything else I thought this was good enough?